initfiles/componentfiles/configschema/xsd/roxie.xsd

```diff
@@ -0,0 +1,640 @@
+                                                    <!--<xs:enumeration value="no" hpcc:description=""/>-->
+                                                <!--</xs:restriction>-->
+                                            <!--</xs:simpleType>-->
+                                        <!--</xs:attribute>-->
+                                        <!--<xs:attribute name="ip" type="ipV4Address" hpcc:displayName="IP Address"-->
+                                                      <!--hpcc:presetValue="0.0.0.0"-->
+                                                      <!--hpcc:tooltip="IP Address of target system"/>-->
+                                        <!--<xs:attribute name="mask" type="ipV4Address" hpcc:displayName="Internet Mask"-->
+                                                      <!--hpcc:presetValue="255.255.255.255"-->
+                                                      <!--hpcc:tooltip="Internet address mask"/>-->
+                                        <!--<xs:attribute name="query" type="xs:string" hpcc:displayName="Query Wildcard"-->
+                                                      <!--hpcc:presetValue=".*"-->
+                                                      <!--hpcc:tooltip="wildcard for queries to allow/disallow"/>-->
+                                        <!--<xs:attribute name="errorCode" type="xs:nonNegativeInteger"-->
+                                                      <!--hpcc:displayName="Error Code"-->
+                                                      <!--hpcc:tooltip="optional error code to associate with the query"/>-->
+                                        <!--<xs:attribute name="error" type="xs:string" hpcc:displayName="Error Message"-->
+                                                      <!--hpcc:tooltip="optional error message to associate with the query"/>-->
+                                        <!--<xs:attribute name="name" type="xs:string" hpcc:displayName="Name"-->
+                                                      <!--use="required" hpcc:autoGenerateType="prefix_"-->
+                                                      <!--hpcc:autoGenerateValue="ACLrule"-->
+                                                      <!--hpcc:uniqueKey="roxieaclaccess_name"-->
+                                                      <!--hpcc:tooltip="Name of this Access Control Rule"/>-->
+                                    <!--</xs:complexType>-->
+                                <!--</xs:element>-->
+                            <!--</xs:sequence>-->
+                            <!--<xs:attribute name="name" type="xs:string" hpcc:displayName="Name" use="required"-->
+                                          <!--hpcc:uniqueKey="roxieacl_name" hpcc:autoGenerateType="prefix_"-->
+                                          <!--hpcc:autoGenerateValue="ACL"/>-->
+                        <!--</xs:complexType>-->
+                    <!--</xs:element>-->
```

Comment:
Should those lines be removed?

Reply:
At this time I am leaving them in place in case the ACLs are not removed from being configured through Roxie. There is a JIRA to remove them and it is desired to do so. However, if they are not removed by the time that config 2.0 goes fully live, these config items will have to be enabled.